### PR TITLE
Feat/#66=create-why-worth-section

### LINKED ISSUE #66: [Landing Page] Create "Dlaczego warto" section
Create the "Dlaczego warto" benefits section (3 illustrated cards) — currently missing from the landing page.

Cards:
1. Zaufanie darczyńców — Darczyńcy wiedzą, że ich pomoc trafia w bezpieczne i uczciwe ręce.
2. Spełnianie realnych potrzeb — Dzięki aktualizowanej na bieżąco liście unikniesz sytuacji niepotrzebnych darów.
3. Proste zarządzanie — Intuicyjny panel zarządzania przyspiesza i ułatwia codzienną pracę.

Figma: https://www.figma.com/design/FBopVCBso8jdxHXzz8wezZ/Potrzebnik-DESKTOP---MOBIEL-Design-system?node-id=4041-13773

**Acceptance criteria:**
- [x] Section renders "Dlaczego warto" heading + 3 cards (illustration, title, description) matching the Figma design
- [x] Responsive on mobile and desktop per Figma specs
- [x] All new components added to Storybook

### PR DESCRIPTION
**Pull request by Prispajer**
Closes #66
 
Adds the "Dlaczego warto" (Why Worth) section to the landing page — three illustrated
cards explaining why it's worth using Potrzebnik.
 
Cards:
- **Donor trust** — Donors know their help lands in safe, honest hands.
- **Real needs fulfilled** — An up-to-date list prevents unnecessary donations.
- **Simple management** — An intuitive management panel speeds up daily work.
 
Desktop: 3-card grid. Mobile: carousel with dots (per Figma).
 
New components (all with Storybook stories): WhyWorthSection, WhyWorthCard,
WhyWorthCarousel, CarouselDots, Stripes, plus the ui/carousel primitive and a dot
variant for Button.
 
Tests: Storybook play tests for the carousel interaction (dot click -> slide,
active-state reporting, single-slide edge case), run against the real browser.
 
Acceptance criteria:
- [x] Heading + 3 cards (illustration, title, description) matching Figma
- [x] Responsive on mobile and desktop
- [x] All new components in Storybook
- [x] Tests for the new components (Storybook)
 
Checks: type-check ✅ · lint ✅ · unit tests ✅ (68/68) · Storybook tests ✅ (67/67) · CI 4/4 ✅

### CODE REVIEW

File: src/app/theme.css [hunk 1]
```diff
@@ -43,6 +43,11 @@
   --color-org-signup-btn-bg: var(--org-signup-btn-bg);
   --color-org-signup-btn-bg-hover: var(--org-signup-btn-bg-hover);
   --color-accent-blue: var(--accent-blue);
+  --color-why-worth-bg: var(--why-worth-bg);
+  --color-why-worth-border: var(--why-worth-border);
```
Comment (marprzybysz) [inline src/app/theme.css]: Looks like another batch of color tokens will need fixing again 🥸

File: package.json
Comment (marprzybysz) [inline package.json]: Won't this package conflict with other packages in the future? 🥸

**Review by marprzybysz — COMMENTED** [review]
Other than my minor questions, I'd say the PR is consistent with the Figma designs, apart from my technical comments regarding backend compatibility 🥸🤓

File: package.json
Comment (pawel-mackiewicz) [reply package.json]: hmm why?

**Comment by LoIlita** [comment]
Checked the branch with current `main` merged. Gates are green locally: type-check, lint, stylelint, prettier, Storybook 67/67. Everything below sits outside what the gates cover.

**Please change**

1. `theme.css:146-148`: `--carousel-dot-bg`, `-hover` and `-active` hold hex literals. Section tokens have to alias a base token, so add the three greys to the base block first and alias them here. This is @marprzybysz's token point. Stylelint misses it because its `theme.css` rule lists section prefixes by name and `carousel-dot` is not on the list. Related: #85, #110.
2. `WhyWorthSection.tsx:19` says "unikasz", #66 says "unikniesz".
3. New stories pin no viewport, so the mobile carousel is never rendered in a test and the section story runs at 1200px, where `xl:` is off. Add Desktop and Mobile stories like `BenefitsStripSection.stories.tsx:21,27`.
4. `WhyWorthSection.tsx:45,55`: the panel starts at `md` but columns only at `xl`, so the three cards stack between 768px and 1280px.
5. UTF-8 BOM at the start of `WhyWorthSection.stories.tsx`, `WhyWorthCard.stories.tsx` and `Stripes.stories.tsx`. #96 stripped those repo wide.
6. `WhyWorthCarousel.tsx:71`: the aria-label reads a slug, "Slajd o nazwie donors". Use `item.title`.
7. `border-[0.32px]` can vanish at DPR 1. The same hairline is `0.5px` elsewhere in the section.
8. `HelpCard.tsx:32` is out of scope for #66, and `aria-label` on `CardTitle` is a prohibited attribute there (it renders a `div`) that repeats the visible text. Please drop it.
9. Dropping `NEXT_PUBLIC_BASE_URL` from `page.tsx` fixes #76. Add `Closes #76` so the issue does not stay open.

**Up for discussion**

10. The AC says "UI only, non-interactive", and this adds a carousel plus `embla-carousel-react`. If Figma requires the carousel, let's correct the issue text instead. `CarouselPrevious` and `CarouselNext` are unused either way.
11. `WhyWorthCard`, `WhyWorthCarousel`, `CarouselDots` and `Stripes` sit in `shared/` with one call site each. Reaching for `shared/` this early is SOLID taken too literally: the folder is for components two or three places genuinely use, and a single-use component there only adds a hop between a section and its own parts. Keep them next to the section until a second caller shows up.
12. The grid and the carousel render the same three cards, so both are in the DOM. Embla also mounts inside `display:none` on desktop and measures zero width.
13. The play test in `WhyWorthCarousel.stories.tsx:32` says it checks scrolling but only asserts `aria-current`.

Text sizes and the other arbitrary values are repo wide, not yours, so they are out of this review and tracked in #89 and #94.

File: package.json
Comment (Prispajer) [reply package.json]: > Won't this package conflict with other packages in the future? 🥸

I think it won't :D

**Comment by LoIlita** [comment]
Gj dude

**Review by LoIlita — APPROVED** [review]
